lib/ci-helper.ts

```diff
@@ -754,7 +755,7 @@ export class CIHelper {
         if (result) {
             const results = await Promise.all(
                 commits.map((commit: IPRCommit) => {
-                const linter = new LintCommit(commit);
+                const linter: LintCommit = new LintCommit(commit);
```

Comment:
Is this really needed?

Reply:
It shouldn't. Of course it should be inferred, don't know why it shows an issue.

Reply:
What linter is saying there is an issue?

Reply:
Seems to be a ReSharper bug.

Without type spec:

![image](https://user-images.githubusercontent.com/4985349/147424965-2ac94f73-3393-45ef-8d1d-a9defee76f80.png)

With type spec:

![image](https://user-images.githubusercontent.com/4985349/147425002-ac879faa-86b6-4937-ac66-ece8f48cc53d.png)

Reply:
have updated the PR to remove those changes.